geo/src/algorithm/convex_hull/mod.rs

```diff
@@ -111,206 +111,4 @@ fn swap_remove_to_first<'a, T>(slice: &mut &'a mut [T], idx: usize) -> &'a mut T
 }
 
 #[cfg(test)]
-mod test {
```

Comment:
unrelated to this failing test, but because this `mod test { ... }` is defined here, the adjacent `test.rs` was previously ignored